spec/s3_meta_sync_spec.rb

```diff
@@ -16,7 +17,8 @@ def upload_simple_structure
   end
 
   def download(file)
-    open("https://s3-us-west-2.amazonaws.com/#{config[:bucket]}/#{file}").read
+    url=URI.parse("https://s3.amazonaws.com/#{config[:bucket]}/#{file}")
```

Comment:
adding open-uri might have already fixed that :)

this works for me:

require 'open-uri'
open('http://google.com').read

On Thu, Sep 10, 2015 at 2:43 PM, Murat Mukhtarov <notifications@github.com>
wrote:

> In spec/s3_meta_sync_spec.rb
> <https://github.com/grosser/s3_meta_sync/pull/12#discussion_r39218020>:
>
> > @@ -16,7 +17,8 @@ def upload_simple_structure
> >    end
> >
> >    def download(file)
> > -    open("https://s3-us-west-2.amazonaws.com/#{config[:bucket]}/#{file}").read
> > +    url=URI.parse("https://s3.amazonaws.com/#{config[:bucket]}/#{file}")
>
> no it doesn't work without URI parser. It would return smth like "file not
> found"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/s3_meta_sync/pull/12/files#r39218020>.
>